opennms-doc/guide-development/src/asciidoc/text/rest/meta-data.adoc

```diff
@@ -0,0 +1,56 @@
+==== Meta-Data
+
+The actual Meta-Data of nodes, interfaces and services can be queried, added, modified or deleted via ReST endpoints.
+
+===== GETs (Reading Meta-Data)
+
+[options="header", cols="5,10"]
+|===
+| Resource                                                                                   | Description
+| `api/v2/nodes/{id}/metaData`                                                               | Get the list of meta-data associated with this node.
```

Comment:
Can we make the paths use lowercase `metadata` instead of `metaData`?

Reply:
done